zanata-overlay/common/bin/start-zanata.sh

```diff
@@ -0,0 +1 @@
+./standalone.sh -c standalone-zanata.xml -P=zanata.properties
```

Comment:
This script won't work unless the user switches to the `$JBOSS_HOME/bin` directory before running it. If we're going to maintain a script for this, it may need to work the same way as `standalone.sh` - see how `$DIRNAME` works in that script.